src/BankingSystem.java

```diff
@@ -0,0 +1,55 @@
+
+public class BankingSystem {
+
+	int creditCount, debitCount, printCount;
+	static int userCrdtCount;
+	static int userDebtCount;
+	static int userprintCount;
+
+	void debitAccount(int debitAmt){
+        debitCount++;
+		userDebtCount++;
+	}
+	void creditAccount(int creditAmt){
+		creditCount++;
+		userCrdtCount++;
```

Comment:
You forgot to perform an operation in debitAcc() and credit() METHODS. You should have a balance instance variable. You should write the condition to verify the insufficient balance.

Reply:
Changes Incorporated.